src/sentry/static/sentry/app/views/onboarding/projectSetup/learnMore.jsx

```diff
@@ -6,21 +6,26 @@ import CreateSampleEventButton from 'app/views/onboarding/createSampleEventButto
 import Panel from 'app/components/panels/panel';
 import PanelBody from 'app/components/panels/panelBody';
 import SentryTypes from 'app/sentryTypes';
+import getDynamicText from 'app/utils/getDynamicText';
 import space from 'app/styles/space';
 
 const LEARN_MORE_VIDEO = 'https://player.vimeo.com/video/319554213';
 
+const learnMoveVideo = getDynamicText({
+  fixed: null,
```

Comment:
I would put placeholder text here so that people looking at the snapshots know there's a placeholder (instead of it looking like the video is just missing)

Reply:
Will open a separate PR to do